src/mainwindow/mainwindow.cpp

```diff
@@ -696,13 +710,45 @@ void MainWindow::restoreMainWindowState()
         restoreState(settings.value("mainWindowState").toByteArray());
 }
 
-void MainWindow::setAppStyle()
+void MainWindow::setInitialAppStyle()
 {
     // fusion style is applied globally
     // if platform does not support fusion, default style is applied
     qApp->setStyle(QStyleFactory::create("Fusion"));
+
+    // restores style palette set in previous app session
+    QSettings savedSettings(QApplication::organizationName(), QApplication::applicationName());
+    QString styleName = {savedSettings.contains("style") ?
+                         savedSettings.value("style").toString()
+                         : "WHITE"};
```

Comment:
it's not obvious that 
`savedSettings.contains("style") ?
                         savedSettings.value("style").toString()
                         : "WHITE"}`
are default settings. comment stm like this

Reply:
added comments